lascli/parser/datasets.py

```diff
@@ -19,6 +44,54 @@ def delete_dataset(las_client: Client, dataset_id, delete_documents):
     return las_client.delete_dataset(dataset_id, delete_documents=delete_documents)
 
 
+def sync(
+    las_client: Client,
+    dataset_id,
+    documents,
+    chunk_size,
+    documents_uploaded,
+    documents_failed,
+    num_threads,
+):
+    log_file = Path(documents_uploaded)
+    error_file = Path(documents_failed)
+    documents = json.loads(Path(documents).read_text())
+    uploaded_files = []
+    counter = Counter()
+
+    if log_file.exists():
+        uploaded_files = log_file.read_text().splitlines()
+
+    if error_file.exists():
+        logging.warning(f'{error_file} exists and will be appended to')
+
+    with log_file.open('a') as lf, error_file.open('a') as ef, ThreadPoolExecutor(max_workers=num_threads) as executor:
```

Comment:
Instead of opening log_file and error_file, how about just using Path's 'write_text()' function?

Reply:
That doesn't work with append